src/Telegram.Bot.Core/ITelegramBotClient.cs

```diff
@@ -24,80 +25,22 @@ public interface ITelegramBotClient
         /// Unique identifier for the bot from bot token. For example, for the bot token
         /// "1234567:4TT8bAc8GHUspu3ERYn-KGcvsvGB9u_n4ddy", the bot id is "1234567".
         /// </summary>
+        [SuppressMessage("ReSharper", "CommentTypo")]
         int BotId { get; }
 
         #region Config Properties
 
-        /// <summary>
-        /// Timeout for requests
-        /// </summary>
-        TimeSpan Timeout { get; set; }
-
-        /// <summary>
-        /// Indicates if receiving updates
-        /// </summary>
-        bool IsReceiving { get; }
-
         /// <summary>
         /// The current message offset
         /// </summary>
         int MessageOffset { get; set; }
 
-        #endregion  Config Properties
-
-        #region Events
-
-        /// <summary>
-        /// Occurs before sending a request to API
-        /// </summary>
-        event EventHandler<ApiRequestEventArgs> MakingApiRequest;
-
-        /// <summary>
-        /// Occurs after receiving the response to an API request
-        /// </summary>
-        event EventHandler<ApiResponseEventArgs> ApiResponseReceived;
```

Comment:
These events should stay. They're meant for debugging purposes and for hooking up into request sending logic.

Reply:
Okay, gonna restore them